src/bridge/GasRefunder.sol

```diff
@@ -216,11 +222,20 @@ contract GasRefunder is IGasRefunder, Ownable {
         uint256 maxRefundeeBalance = commonParams.maxRefundeeBalance;
         uint256 maxSingleGasUsage = commonParams.maxSingleGasUsage;
 
+        // add any cost for 4844 data, the data hash reader throws an error prior to 4844 being activated
+        try dataHashReader.getDataHashes() returns (bytes32[] memory dataHashes) {
+            if (dataHashes.length != 0) {
+                uint256 blobBasefee = blobBasefeeReader.getBlobBaseFee();
+                gasUsed += dataHashes.length * gasPerBlob * blobBasefee;
```

Comment:
Shouldn't we `/ block.basefee` since gasUsed is measured in gas not wei?

Reply:
yes, good catch!